scripts/ci/horti.env

```diff
@@ -0,0 +1,5 @@
+MARKET_URL_READ=https://staging.dev.medicmobile.org
+BUILDS_SERVER=_couch/builds_testing
+BUILD_NUMBER=1221218955
+NODE=14.x
+TAG=3.11.1-beta.1
```

Comment:
Are all these values placeholders?

Reply:
Yeah they are for local testing. So you can do `docker-compose -f scripts/ci/horti_compose.yml --env-file scripts/ci/horti.env up ` for testing locally. I was conflicted on whether or not to commit this but felt it might help when troubleshooting.

Reply:
Can we please then use a Node version that we support? :) 
Also, please add some comments about what these are.